src/pages/networks/NetworkDetail.tsx

```diff
@@ -47,15 +50,15 @@ const NetworkDetail: FC = () => {
   }
 
   const isManagedNetwork = network?.managed;
+  const isFeatureSupported =
```

Comment:
```suggestion
  const hasForwards =
```
I think we should only use this for forwards and have another variable for leases.